readme.md

```diff
@@ -219,27 +220,28 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
   const { generate: generateId } = require('shortid')
   ```
 
-* **Your Answer:**
+* **Your Answer:** ShortId creates short non-sequential unique ids. The ^ code creates a shortid. The syntax is called 
+destructuring and in thsi case it matches objects.
 
 * **Question:** What routes are currently available to you as part of this application?
 
-* **Your Answer:**
+* **Your Answer:** /vegetables and /vegetables/:id and it also specifies if it is doesnt not match /vegtables, to return a 404 with `Could not find vegetable with ID of ${id}`.
 
 * **Question:** Look for `helpers.validate` in the `app.js` file. What is this and how does it work?
 
-* **Your Answer:**
+* **Your Answer:** This validates the object/variable you defined. In this case helpers is a imported module. The .validate() function returns an array of validation errors.
 
 * **Question:** Try creating a new vegetable. Then, try restarting your server. What happens to the data you posted and why?
 
-* **Your Answer:**
```

Comment:
Data does not persist after sever is restarted since it is not saved to a file or database